lib/utils/lifecycle.js

```diff
@@ -262,6 +262,28 @@ function runCmd_ (cmd, pkg, env, wd, stage, unsafe, uid, gid, cb_) {
     sh = process.env.comspec || 'cmd'
     shFlag = '/d /s /c'
     conf.windowsVerbatimArguments = true
+
+    // check if we have a composed command
+    // e.g.: `"test": "node ../test/test"`
+    if (cmd.indexOf(' ') !== -1) {
+      // split the command
+      var cmdChunks = cmd.split(' ')
+
+      cmd = cmdChunks.map(function (chunk) {
+        // let's assume that a path starts with either `./` or `../`
```

Comment:
Well under Windows `/my/folder/and/file.js` doesn't exist. If you wanna use an absolute path you could only do `C:\\my\\folder\\and\\file.js`.

Regarding the relative path my aim is not to change the behaviour of npm. 

The question about does the file exist or not will be managed by the `spawn` [here](https://github.com/npm/npm/pull/16498/files#diff-8c44327c5a50975dee6ded9e9789e96eR293)